src/Entity/Petition.php

```diff
@@ -49,6 +49,32 @@ class Petition
     #[ORM\Column(length: 255, unique: true)]
     private string $public_id;
 
+    /**
+     * @param bool $closed
+     * @param string $title
+     * @param string $subtitle
+     * @param string $text
+     * @param string $target
+     * @param string $target_to_whom
+     * @param string $author
+     * @param string $author_to_whom
+     * @param string $author_birthdate
+     * @param string $public_id
+     */
+    public function __construct(bool $closed, string $title, string $subtitle, string $text, string $target, string $target_to_whom, string $author, string $author_to_whom, string $author_birthdate, string $public_id)
```

Comment:
let's revert this change and use existing set-methods instead of this constructor